### ISSUE TITLE
Can't Show the datepicker on .open() function

### ISSUE THREAD

**Issue by AndryHTC** [benitolopez/hotel-datepicker#35]
The datepicker work perfectly, but doesn't appear if I apply the function .open() to the HotelDatepicker object, by clicking on the element I want. The click event is recorded in the console, so the problem is the hotel-datepicker object. No JS error in the console

```
picker_options = {
    autoClose: true,
    startDate: today,
    format: 'DD-MM-YYYY',
    startOfWeek: 'monday',
    moveBothMonths: true,
    separator: ' / '
}
var Hotel_Datepicker = new HotelDatepicker(datepicker_input_DOM, picker_options);

$(dom_selected_element).on('click',function(){
     console.log("CLICKED");
     Hotel_Datepicker.open();
});
```

**Comment by benitolopez** [comment]
Tried right now and I don't have any problem. Check your code because there's an error. 

`);` after the last `}`

```js
$(dom_selected_element).on('click',function(){
    console.log("CLICKED");
    Hotel_Datepicker.open();
});
```

You can also use Javascript:

```js
var button = document.getElementById('your-button-id');

button.addEventListener('click', your_custom_function);

function your_custom_function() {
	console.log('clicked');
	datepicker.open();
}
```

**Comment by AndryHTC** [comment]
I've copied wrong the code from the editor... Don't worry, JS go into that function because it prints to the console "CLICKED", so that isn't the problem. I've also tried with pure JS instead of jQuery, but same result.

No error come out on the console. But If I click on the datepicker_input_DOM, after clicking many times on the dom_selected_element, come up Datepickers as many as I clicked on dom_selected_element; so then when I want to close it, I have to close all the datepickers (that were appeared one below the others).
